src/game-support/hm2-wn.md

```diff
@@ -0,0 +1,6 @@
+# Hotline Miami 2: Wrong Number
+<!-- script:Aliases [] -->
```

Comment:
Please add the following aliases:
```
<!-- script:Aliases [
    "Hotline Miami 2",
    "Hotline Miami 2 Wrong Number",
    "Hotline Miami II",
    "Hotline Miami II Wrong Number",
    "Hotline Miami II: Wrong Number"
] -->
```

Reply:
Okay sure thing, I just made the changes. :)